core/i18n/resources/en.js

```diff
@@ -402,6 +403,9 @@ $t(common.cantUndoWarning)`,
 {{rStudioCode}}
 
 `,
+    entities: {
+      new: 'Virtual entity',
+    },
```

Comment:
can this just be virtualEntity? instead of entities.new

Reply:
I added this because maybe we need to add more things, but yes!